mu-plugins/blocks/time/postcss/style.pcss

```diff
@@ -0,0 +1,3 @@
+.wporg-time {
+	text-decoration: underline dotted;
```

Comment:
I find it's useful to be able to see the content the format has been applied to, and this seems the most appropriate style given that it will end up as a link on the frontend.

Reply:
The block editor content is rendered in an iframe. I don't think this will work.

Reply:
Does it not show for you?

<img src="https://github.com/WordPress/wporg-mu-plugins/assets/1017872/1a5810b4-984a-461f-ae61-ed038703340f" width="500">

Reply:
I see it in the post editor and page editor, but not in the site editor. In fact there are issues with the Gutenberg selectors in the site editor too. I've opened a follow up [issue](https://github.com/WordPress/wporg-mu-plugins/issues/422).

Reply:
I've added partial support in the Site Editor by using inline styles https://github.com/WordPress/wporg-mu-plugins/pull/415/commits/ea1dad6132df73bf94f494d31bf59dee379b5267

I'll do further work in the follow up issue once I have a solution.